geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -69,6 +78,17 @@
    */
   private static final String KEY_SLOW_START_TIME_FOR_TESTING = "slowStartTimeForTesting";
 
+
+  /**
+   * Default value for waitinig for re-authentication
+   */
+  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = 5000;
+
+  /**
+   * Key in the system property from which the slow starting time value will be retrieved
+   */
+  protected static final String RE_AUTHENTICATE_WAIT_TIME_KEY = "geode.reauthenticate.wait.time";
```

Comment:
Consider using SystemPropertyHelper for you new system properties. It has support for both prefixes (geode. or gemfire.).